src/db/util.ts

```diff
@@ -0,0 +1,7 @@
+export const randomNumericRating = () => Math.floor(Math.random() * 12) / 2;
```

Comment:
thought we said we were gonna update this to allow `null` ?

Reply:
o yea